akka-stream-tests/src/test/scala/akka/stream/impl/FixedBufferSpec.scala

```diff
@@ -6,7 +6,9 @@ package akka.stream.impl
 
 import akka.stream.testkit.StreamSpec
 import akka.stream.ActorMaterializerSettings
+import com.github.ghik.silencer.silent
 
+@silent("deprecated")
```

Comment:
Not switching to attributes for this test?

Reply:
This test actually verifies the MaterializerSettings behavior, which we are still keeping until 2.7.